src/app/parcels/getExpandedParcelDetails.ts

```diff
@@ -138,72 +151,6 @@ export const formatDatetimeAsTime = (datetime: string | null): string => {
     return new Date(datetime).toLocaleTimeString("en-GB", { hour: "2-digit", minute: "2-digit" });
 };
 
-export const formatAddressFromClientDetails = (
-    client: Pick<
-        Schema["clients"],
-        "address_1" | "address_2" | "address_town" | "address_county" | "address_postcode"
-    >
-): string => {
-    return [
-        client.address_1,
-        client.address_2,
-        client.address_town,
-        client.address_county,
-        client.address_postcode,
-    ]
-        .filter((field) => field)
-        .join(", ");
-};
-
-export const formatHouseholdFromFamilyDetails = (
-    family: Pick<Schema["families"], "age" | "gender">[]
-): string => {
-    let adultCount = 0;
-    let childCount = 0;
-
-    for (const familyMember of family) {
-        if (familyMember.age === null || familyMember.age >= 16) {
-            adultCount++;
-        } else {
-            childCount++;
-        }
-    }
-
-    const adultChildBreakdown = [];
-
-    if (adultCount > 0) {
-        adultChildBreakdown.push(`${adultCount} adult${adultCount > 1 ? "s" : ""}`);
-    }
-
-    if (childCount > 0) {
-        adultChildBreakdown.push(`${childCount} child${childCount > 1 ? "ren" : ""}`);
-    }
-
-    const familyCategory = familyCountToFamilyCategory(family.length);
-    const occupantDisplay = `Occupant${adultCount + childCount > 1 ? "s" : ""}`;
-
-    return `${familyCategory} ${occupantDisplay} (${adultChildBreakdown.join(", ")})`;
-};
-
-export const formatBreakdownOfChildrenFromFamilyDetails = (
-    family: Pick<Schema["families"], "age" | "gender">[]
-): string => {
-    const childDetails = [];
-
-    for (const familyMember of family) {
-        if (familyMember.age !== null && familyMember.age <= 15) {
-            const age = familyMember.age === -1 ? "0-15" : familyMember.age.toString();
-            childDetails.push(`${age}-year-old ${familyMember.gender}`);
-        }
-    }
-
-    if (childDetails.length === 0) {
-        return "No Children";
-    }
-
-    return childDetails.join(", ");
-};
```

Comment:
The exact copy of these functions already exist in `getExpandedClientDetails.ts`